WalletWasabi.Gui/Controls/WalletExplorer/SendControlViewModel.cs

```diff
@@ -871,63 +867,67 @@ private bool TryParseUserFee(out decimal userFee)
 				&& userFee > 0;
 		}
 
-		public ErrorDescriptors ValidateUserFeeText()
+		private void ValidateUserFeeText(IErrorList errors)
 		{
-			return TryParseUserFee(out _)
-				? ErrorDescriptors.Empty
-				: new ErrorDescriptors(new ErrorDescriptor(ErrorSeverity.Error, "Invalid fee."));
+			if (!TryParseUserFee(out _))
+			{
+				errors.Add(ErrorSeverity.Error, "Invalid fee.");
+			}
 		}
 
-		public ErrorDescriptors ValidatePassword() => PasswordHelper.ValidatePassword(Password);
+		private void ValidatePassword(IErrorList errors) => PasswordHelper.ValidatePassword(errors, Password);
 
-		public ErrorDescriptors ValidateActiveAddress()
+		private void ValidateAddress(IErrorList errors)
 		{
 			if (string.IsNullOrWhiteSpace(Address))
 			{
-				return ErrorDescriptors.Empty;
+				return;
 			}
 
-			if (Address == CustomChangeAddress)
+			if (!string.IsNullOrWhiteSpace(CustomChangeAddress))
```

Comment:
> Settings tab doesn't open:
> 
> ```
> 2020-04-27 03:04:14 ERROR       ToolCommands (47)       System.ArgumentException: Cannot bind to the target method because its signature is not compatible with that of the delegate type.
>    at System.Reflection.RuntimeMethodInfo.CreateDelegateInternal(Type delegateType, Object firstArgument, DelegateBindingFlags bindingFlags)
>    at System.Reflection.RuntimeMethodInfo.CreateDelegate(Type delegateType, Object target)
>    at WalletWasabi.Gui.ViewModels.ViewModelBase.RegisterValidationMethods() in C:\Users\RESEAU\Desktop\WalletWasabi\WalletWasabi.Gui\ViewModels\ViewModelBase.cs:line 46
>    at WalletWasabi.Gui.ViewModels.ViewModelBase..ctor() in C:\Users\RESEAU\Desktop\WalletWasabi\WalletWasabi.Gui\ViewModels\ViewModelBase.cs:line 23
>    at WalletWasabi.Gui.ViewModels.WasabiDocumentTabViewModel..ctor(String title) in C:\Users\RESEAU\Desktop\WalletWasabi\WalletWasabi.Gui\ViewModels\WasabiDocumentTabViewModel.cs:line 20
>    at WalletWasabi.Gui.Tabs.SettingsViewModel..ctor() in C:\Users\RESEAU\Desktop\WalletWasabi\WalletWasabi.Gui\Tabs\SettingsViewModel.cs:line 44
>    at WalletWasabi.Gui.Shell.Commands.ToolCommands.<>c.<.ctor>b__0_4() in C:\Users\RESEAU\Desktop\WalletWasabi\WalletWasabi.Gui\Shell\Commands\ToolCommands.cs:line 30
>    at AvalonStudio.Shell.IShellExtensions.AddOrSelectDocument[T](IShell me, Func`1 factory)
>    at WalletWasabi.Gui.Shell.Commands.ToolCommands.<>c.<.ctor>b__0_0() in C:\Users\RESEAU\Desktop\WalletWasabi\WalletWasabi.Gui\Shell\Commands\ToolCommands.cs:line 30
>    at ReactiveUI.ReactiveCommand.<>c__DisplayClass0_0.<Create>b__1(IObserver`1 observer) in d:\a\1\s\src\ReactiveUI\ReactiveCommand\ReactiveCommand.cs:line 108
>    at System.Reactive.Linq.QueryLanguage.CreateWithDisposableObservable`1.SubscribeCore(IObserver`1 observer) in D:\a\1\s\Rx.NET\Source\src\System.Reactive\Linq\QueryLanguage.Creation.cs:line 35
>    at System.Reactive.ObservableBase`1.Subscribe(IObserver`1 observer) in D:\a\1\s\Rx.NET\Source\src\System.Reactive\ObservableBase.cs:line 58
> ```

Fixed!

Reply:
Check removed.